File: Computabilidade-e-Complexidade-de-Algoritmos-main/golang/13.go
package main
import "fmt"

func maisUns(s string) bool {
    count1, count0 := 0, 0
    for _, char := range s {
        if char == '1' {
            count1++
        } else if char == '0' {
            count0++
        }
    }
    return count1 > count0
}

func main() {
    fmt.Println(maisUns("110"))    // True
    fmt.Println(maisUns("10100"))  // False
}

File: Computabilidade-e-Complexidade-de-Algoritmos-main/java/13.java
public class AFDMaisUns {
    public static boolean maisUns(String s) {
        int count1 = 0, count0 = 0;
        for (char c : s.toCharArray()) {
            if (c == '1') count1++;
            else if (c == '0') count0++;
        }
        return count1 > count0;
    }

    public static void main(String[] args) {
        System.out.println(maisUns("110"));    // True
        System.out.println(maisUns("10100"));  // False
    }
}
